lib/stub_ui/lib/geometry.dart

```diff
@@ -194,7 +194,7 @@ class Offset extends OffsetBase {
   // This is included for completeness, because [Size.infinite] exists.
   static const Offset infinite = const Offset(double.infinity, double.infinity);
 
-  /// Returns a new offset with the x component scaled by `scaleX` and the y
+  /// Returns a offset with the x component scaled by `scaleX` and the y
```

Comment:
new should not be removed inside comments :)

Reply:
Ah thanks.  There were a few places where it made sense in the comments for sample code, guess I was overzealous

Reply:
Leaving stub_ui as is, since there's another PR to remove all those comments anyway